### PR TITLE
Jc leo

### PR DESCRIPTION
**Pull request by jchanbroad**
\<your comments for this PR go here\>

Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/leonardo/blob/develop/CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get a thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: automation/src/test/scala/org/broadinstitute/dsde/workbench/NotebooksCanaryTest.scala [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      val localizeDataContents = "Hello World"
+
+      val project = GoogleProject("automated-notebooks-canary")
+
+      withNewCluster(project, monitorDelete = true, apiVersion = V2) { cluster =>
+        val clusterStatus = cluster.status
+        withWebDriver { implicit driver =>
+          withNewNotebook(cluster, Python3) { notebook =>
+            print(notebook.executeCell("""print("hi")""") shouldBe (Some("hi")))
+            println(clusterStatus shouldBe(ClusterStatus.Running))
```
Comment (MatthewBemis) [inline automation/src/test/scala/org/broadinstitute/dsde/workbench/NotebooksCanaryTest.scala]: i think the `println` here should be removed

File: automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoConfig.scala [hunk 2]
```diff
@@ -8,6 +8,8 @@ object LeonardoConfig extends CommonConfig {
   object Leonardo {
     val apiUrl: String = leonardo.getString("apiUrl")
     val notebooksServiceAccountEmail: String = leonardo.getString("notebooksServiceAccountEmail")
+    val notebooksCanaryProject: String = leonardo.getString("notebooksCanaryProject")
+    //val notebooksCanaryProjectProd: String = leonardo.getString("notebooksCanaryProjectProd")
```
Comment (MatthewBemis) [inline automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoConfig.scala]: is this meant to be commented out?

File: automation/src/test/scala/org/broadinstitute/dsde/workbench/NotebooksCanaryTest.scala [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+import org.broadinstitute.dsde.workbench.leonardo.{cluster, _}
+
+
+class NotebooksCanaryTest extends FreeSpec with Matchers with NotebookTestUtils with ParallelTestExecution with
+  BillingFixtures {
+
+
+
+  implicit val authToken: AuthToken = ronAuthToken
+  "Test for creating a cluster and localizing a notebook" - {
```
Comment (MatthewBemis) [inline automation/src/test/scala/org/broadinstitute/dsde/workbench/NotebooksCanaryTest.scala]: nit: i wonder if this should be called something like "Leonardo Canary Test" and the following block, which contains the actual test, can be the description of what should happen

**Review by MatthewBemis — COMMENTED** [review]
just a few minor comments! overall looks good

**Comment by jchanbroad** [comment]
jenkins rebuild

**Comment by jchanbroad** [comment]
jenkins retest

**Comment by jchanbroad** [comment]
jenkins rebuild

**Comment by jchanbroad** [comment]
jenkins retest

**Comment by jchanbroad** [comment]
jenkins retest

**Comment by jchanbroad** [comment]
jenkins retest

**Comment by jchanbroad** [comment]
travis retest

**Comment by jchanbroad** [comment]
jenkins retest

**Comment by andy7i** [comment]
jenkins retest

**Comment by jdcanas** [comment]
Closing this as Jocelyn's co-op has ended and this has diverged pretty far. There is a (currently unprioritized) ticket here for its continuation: https://broadworkbench.atlassian.net/browse/IA-1271

**Pull request closed without merging**